migration/mule/scatter-gather-flow-control/ScatterGatherFlowControl/src/main/synapse-config/api/ScatterGatherFlowControl.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<api context="/aggregate" name="ScatterGatherFlowControl" xmlns="http://ws.apache.org/ns/synapse">
+    <resource methods="GET">
+        <inSequence>
+            <property action="remove" name="NO_ENTITY_BODY" scope="axis2"/>
+            <clone id="aggregate_id">
+                <target>
+                    <sequence>
+                        <fileconnector.read>
+                            <source>[file path of contacts-1.csv]</source>
```

Comment:
How do we skip reading the header line (first line in the .csv file)?

Reply:
This is fixed by providing numberOfLinesToSkip.